lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -586,7 +586,9 @@ sub to_hash {
         $j = {%$j, %{$job->deps_hash}};
     }
     if ($args{details}) {
-        $j->{testresults} = read_test_modules($job);
+        $j->{testresults}      = read_test_modules($job);
```

Comment:
Perhaps for consistency read_test_modules could be also moved as part of the model and not from utility?